pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -675,6 +676,10 @@ type APIServerSettings struct {
 	NodePortRange               string `json:"nodePortRange,omitempty"`
 }
 
+type KonnectvityProxySettings struct {
+	DeploymentSettings `json:",inline"`
+}
```

Comment:
Neither replicas nor tolerations from `DeploymentSettings` are respected here because konnectivity is not a `Deployment` but a sidecar. Shouldn't this be this instead?

```suggestion
type KonnectvityProxySettings struct {
	Resources   *corev1.ResourceRequirements `json:"resources,omitempty"`
}
```

Reply:
Definitely. Thanks!